src/components/_common/matchCardTest.vue

```diff
@@ -0,0 +1,18 @@
+<script lang="ts">
+import { PropType } from 'vue';
+import type { Match } from '../../types/match';
+export default {
+  props: {
+    match: {
+      type: Object as PropType<Match>,
+      required: true,
+    },
+  },
+};
+</script>
+
+<template>
+  <div class="flex flex-col p-2">
+    {{ match?.game }}
+  </div>
+</template>
```

Comment:
We tend to keep SFCs in Vue PascalCase for their file name (e.g. `MatchCardTest.vue`)